content/blog/vscode-esp-idf-v1.8.0/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Visual Studio Code ESP-IDF v1.8.0 extension"
+date: 2024-07-12T16:07:09+08:00
+showAuthor: false
+tags: ["News", "ESP-IDF extension", "VSCode", "Visual Studio Code"]
+authors:
+  - "brian-ignacio"
+---
+
+## Introduction
+
+With the v1.8.0 of [Espressif ESP-IDF for Visual Studio Code extension](https://github.com/espressif/vscode-esp-idf-extension) we introduce a few new features and lot of bug fixes as described in [v1.8.0 Release](https://github.com/espressif/vscode-esp-idf-extension/releases/tag/v1.8.0). If you face any issue or improvement ideas please share them as a [Github issue](https://github.com/espressif/vscode-esp-idf-extension/issues).
```

Comment:
It might be better to use VS Code in the extension name and elsewhere, because it is shorter and almost everybody know that it stands for Visual Studio Code anyway.